Python/Question_05/Challenge-05.py

```diff
@@ -0,0 +1,31 @@
+def pattern5(level):
+    '''This function prints the following pattern:
+      *
+     ***
+    *****
+    
+    But the output of the program is 
+    Enter the level: 5
+*
+     ***
+          *****
+               *******
+                    *********
+                 
+find the bug and solve accordingly                 
+    '''
```

Comment:
You can have this before the function starts. You can also show the question in the README.md file.